Earthfile

```diff
@@ -130,9 +136,60 @@ build-yawollet-image:
       -var "network_id=$OS_NETWORK_ID" \
       -var "floating_network_id=$OS_FLOATING_NETWORK_ID" \
       -var "security_group_id=$OS_SECURITY_GROUP_ID" \
+      -var "machine_flavor=$MACHINE_FLAVOR" \
+      -var "volume_type=$VOLUME_TYPE" \
       -var "image_visibility=$IMAGE_VISIBILITY" \
       image/alpine-yawol.pkr.hcl
 
+build-packer-environment:
+    FROM +terraform
+
+    ARG FLOATING_NETWORK_NAME=floating-net
+
+    ARG --required OS_AUTH_URL
+    ARG --required OS_PROJECT_ID
+    ARG --required OS_PROJECT_NAME
+    ARG --required OS_USER_DOMAIN_NAME
+    ARG --required OS_PASSWORD
+    ARG --required OS_USERNAME
+    ARG --required OS_REGION_NAME
+
+    COPY --dir hack/packer-infrastructure .
+
+    WORKDIR /packer-infrastructure
+
+    RUN terraform init
+    RUN terraform apply \
+      -var "floating_ip_network_name=$FLOATING_NETWORK_NAME" \
+      -auto-approve
+
+    SAVE ARTIFACT /packer-infrastructure/terraform.tfstate AS LOCAL hack/packer-infrastructure/terraform.tfstate
+
+destroy-packer-environment:
+    FROM +terraform
+
+    ARG FLOATING_NETWORK_NAME=floating-net
```

Comment:
do we want to have these in the repo? guess these "sensible defaults" only apply to stackit right?